Nodejs/Product/Nodejs/Nodejs.csproj

```diff
@@ -1597,6 +1604,7 @@
     </ItemGroup>
   </Target>
   <PropertyGroup>
-    <PostBuildEvent>copy /y $(OutDir)..\WebRole\Microsoft.NodejsTools.WebRole.dll $(OutDir)Debugger\RemoteDebug\</PostBuildEvent>
+    <PostBuildEvent>echo f | xcopy /y $(ProjectDir)Templates\Files\DockerfileTemplate\Dockerfile $(OutDir)Templates\Files\DockerfileTemplate\Dockerfile

```

Comment:
xcopy will create new directories (unlike copy), but also asks whether it's a file or a directory (f or d), and there is no other way to suppress this.